eos-tech-support/eos-diagnostics

```diff
@@ -127,23 +127,35 @@ function collectTemperatureInfo() {
 }
 
 function collectChromiumPluginsInfo() {
-    let output = 'Version: ';
-
+    let flashOutput = 'Flash Version: ';
     let flashPath = Gio.file_new_for_path('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libpepflashplayer.so');
-    if (!flashPath.query_exists(null)) {
-	output += 'not installed\n';
-	return output;
+    if (flashPath.query_exists(null)) {
+        try {
+	    let [res, version] = GLib.file_get_contents('/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/Flash_VERSION.txt');
```

Comment:
I guess everyone with this new version of `eos-diagnostics` will have run the new updater, so there's no need to fall back to the old `VERSION.txt` file?